content/educators/thomas.md

```diff
@@ -0,0 +1,9 @@
+---
+name: Thomas
+pronouns_en: he/him
+pronouns_nl: hij/hem
+quote:
+  Door in gesprek te gaan en mezelf te zijn, hoop ik anderen te laten zien dat dit ook oké is!
+text:
```

Comment:
grappig, volgens mij doen we helemaal niks met deze tekst, maar zie wel dat alle voorlichters het hebben 😅

Reply:
Ja idd, maar ik dacht ik hou het er maar gewoon in, just in case dat we dat ooit willen veranderen